tests/Temporalio.Tests/Client/TemporalClientTests.cs

```diff
@@ -48,4 +52,124 @@ public async Task CreateLazy_Connection_NotConnectedUntilCallMade()
         Assert.True(client.Connection.IsConnected);
         client.Connection.RpcMetadata = new Dictionary<string, string>();
     }
+
+    [Fact]
+    public async Task ConnectAsync_Connection_AllGrpcCallsSupported()
+    {
+        // The approach we'll take here is to just start the dev server and reflectively make each
+        // call in workflow service and operator service and check metrics to confirm that every
+        // call got that far into core. This catches cases where we didn't add all string call names
+        // in Rust for our C# methods.
+        var captureMeter = new CaptureRpcCallsMeter();
+        await using var env = await Temporalio.Testing.WorkflowEnvironment.StartLocalAsync(new()
+        {
+            Runtime = new(new()
+            {
+                Telemetry = new() { Metrics = new() { CustomMetricMeter = captureMeter } },
+            }),
+        });
+
+        // Check workflow service and operator service
+        await AssertAllRpcsAsync(captureMeter.Calls, env.Client.Connection.WorkflowService);
+        await AssertAllRpcsAsync(captureMeter.Calls, env.Client.Connection.OperatorService, skip: "AddOrUpdateRemoteCluster");
```

Comment:
Curious why this call is skipped and if this is temporary.

Reply:
It hangs/messes up the server when you call it with an empty request